projekt-03-kontakty-main/projekt-03-kontakty-main/style.css

```diff
@@ -8,3 +8,44 @@ body {
 }
 
 /* ---------------------------------------------------- */
+
+.container {
+	display: inline-flexbox;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-content: center;
+	background-color: whitesmoke;
+	border-radius: 1rem;
+	align-items: stretch;
+}
+
+.item {
+	display: inline-block;
+	max-width: 22%;
+	padding: 2rem;
+	margin: 2rem;

```

Comment:
Tady je vidět, že nám odsazení mezi 2 itemy nevychází. Je to proto, že nastavuješ margin všude - i zleva i zprava. Takže tenhle margin se ti mezi prvky sečte a je dvojnásobný. Zase existuje víc způsobů, jak to řešit. Například na kontejneru nastavit padding - tím vyřešíme všechny vnější okraje. Potom itemům nastavit jen margin zprava.
Tím nám ale vznikne problém s posledním itemem, který bude mít margin zprava + padding kontejneru. Můžeme to vyřešit trochu složitějším CSS selektorem, kdy vybereme všechny prvky kromě posledního a jen těm nastavíme margin zprava:
```
.item:not(:last-child) {
	margin-right: 2rem;
}

```

Reply:
Ještě jednodušší varianta by bylo nastavit margin na 1rem a padding na kontejneru .container zprava a zleva taky na 1rem, a mělo by to taky sedět